src/section_info/mod.rs

```diff
@@ -70,6 +73,9 @@ pub enum GetSectionResponse {
     /// Successful response to `GetSectionQuery`. Contains information about the requested
     /// section.
     Success(SectionInfo),
+    /// The node is not allowed to join the network
+    /// since it it not externally reachable
+    NotAllowed,
```

Comment:
Should this be `NotReachable` ?

Reply:
`SectionInfoResponse::NotReachable` didn't seem very coherent to the request that's being sent.
In the future, we could expand `NotAllowed(Reason::NotReachable)` if this is being used for multiple reasons. But for now this should be okay I think?

Reply:
I don't think it is clear to readers why we are denying access though. I think specific here is maybe best?

Reply:
Or `SectionInfoResponse::NodeNotReachable`

Reply:
Okay. I'll make that update then :+1: